sdk/test/UnitTests/HttpWebRequestTracingExtensionTests.cs

```diff
@@ -217,5 +217,32 @@ public async Task TestExceptionGetAsyncResponseTraced()
             }
         }
 #endif
+
+        [TestMethod]
+        public async Task TestContextMissingStrategyGetAsyncResponseTraced()
+        {
+            _recorder = new AWSXRayRecorder();
+#if NET45
+            AWSXRayRecorder.InitializeInstance(_recorder);
+#else
+            AWSXRayRecorder.InitializeInstance(recorder: _recorder);
+# endif
+            Assert.IsFalse(AWSXRayRecorder.Instance.IsTracingDisabled());
+
+            AWSXRayRecorder.Instance.ContextMissingStrategy = Core.Strategies.ContextMissingStrategy.LOG_ERROR;
+
+            _recorder.EndSegment();
+
+            // The test should not break. No segment is available in the context, however, since the context missing strategy is log error,

```

Comment:
Is there a way to detect if the logger prints the correct message?

Reply:
Thats interesting point. However, there is no way to get the value from the logger message at the moment. I think, in future, we need to have custom logger implementation which has access to logger instance at AWSXRayRecorder level, to get the value of the message.

Reply:
Any chance we can rerun now that we have appveyor? :)

Reply:
Sure, done.